### ISSUE TITLE
image options

### ISSUE THREAD

**Issue by ceheitsch** [gtDMMB/RNAStructViz#99]
It looks like there are more image options than just PNG...?  If so, they should be described (briefly) on the wiki; there isn't enough space in the application note.  Also, how extensively have these been tested?

**Comment by maxieds** [comment]
@ceheitsch 
There are tooltips on almost all of the widget buttons in the StructViz GUI. What this second icon does (you can click on them as well) is specify whether or not the currently loaded file should be auto re-loaded the next time the application starts -- effectively pinning that structure file to the user's workspace until it is deselected. This is actually a desirable feature since restarting the application, or picking out a preset list of structures to load can be troublesome if they are in different directories, or if the CT/NOPCT files come from a large folder of data that is hard to navigate:

**(Tooltip documenting the "hidden" feature)**

![tooltip](https://user-images.githubusercontent.com/22165688/89065696-288b4080-d33a-11ea-8c0d-dc7c7aa1195f.png)

**(Demo of re-loading the structures at next launch)**

![reloading-autoloaded-struct-files](https://user-images.githubusercontent.com/22165688/89065694-288b4080-d33a-11ea-95c6-1ca4adfb7f8d.png)

Note that taking these screenshots did bring up an unanticipated bug in that the last release will only load one file per base sequence at a time. This will be fixed when I issue a new release, most likely by the end of today.

**Comment by maxieds** [comment]
@ceheitsch 

There are examples of output in all the formats we support (from images to CSV plaintext from the stats window) [located here for documentation](https://github.com/gtDMMB/RNAStructViz/tree/master/example-outputs). It hasn't been tested by me recently, but the last time I picked this up I created the sample outputs file. Thus I am confident that these export image formats do what they are supposed to (and look consistent). 

The SVG image output was actually a feature that was brought to my attention as wanted when we did the peer group testing rounds. SVG is portable and a non-proprietary format that is standard across the systems we support. The C-HEADER format is more of something that I wanted to make things easier for development. This is essentially in the same GIMP-inspired format as you will see in [this directory](https://github.com/gtDMMB/RNAStructViz/tree/master/src/pixmaps). This pixel buffer based header format is the way we can include high resolution images, icons and logos with FLTK without having to resort to storing them somewhere on the end user's filesystem (the buffers are static and get compiled into the binary).

I will work on documenting these format options on the WIKI after more coding for the other issues today.
